jdr/src/main/java/org/jboss/as/jdr/plugins/AS7Plugin.java

```diff
@@ -45,8 +45,10 @@ public List<JdrCommand> getCommands() throws Exception {
             new CollectFiles("*.properties").sanitizer(passwordSanitizer),
             new CollectFiles("*.xml").sanitizer(xmlSanitizer),
             new CollectFiles("*/modules/system/*/.overlays/.overlays"),
-            new CollectFiles("*/.installation/*.conf"),
+            new CollectFiles("*/.installation/*.conf"), // TODO there are no such files
             new CollectFiles("*/.installation/*.txt"),
+            new CollectFiles("*/.installation/*.yaml"),
+            new CollectFiles("*/.installation/*.yml"), // currently none without the 'a', but having this makes collection more future proof
```

Comment:
it might make sense to include `.installation/*.xml` to capture `installation/provisioning_record.xml` and `.galleon/*.xml`

Reply:
@spyrkob Yes, they are already included.